NEW_README.md

```diff
@@ -143,84 +319,129 @@ to the version of PE you want to compare against.
 
 Things to note:
 
-* If BASELINE_PE_VER has not been set than the baseline comparison assertions in the last test step will not be run (test step will be skipped).
-* Whatever string you pass to the job to indicate the BEAKER_PE_VER needs to be used consistently. For example, if you set the version in the format `2018.1.1-rc0-11-g8fbde83` you need to use this again as the BASELINE_PE_VER or the test will error/not return expected results.
-* If BASELINE_PE_VER is not found in BigQuery then the last step will error.
-* Results are not overwritten, we get the latest result that matches BASELINE_PE_VER and the current test name.
+* If `BASELINE_PE_VER` has not been set than the baseline comparison assertions in the last test step will not be run (test step will be skipped).
+
+* Whatever string you pass to the job to indicate the `BEAKER_PE_VER` needs to be used consistently. For example, if you set the version in the format `2018.1.1-rc0-11-g8fbde83` you need to use this again as the `BASELINE_PE_VER` or the test will error/not return expected results.
```

Comment:
@billclaytor  any idea what this is even saying?  I think it is bogus.  pe ver is what you are running... it is what will get set in bigquery if you push results... but what is it supposed to be consistent with?  The only thing I can think of is the value in BASELINE_PE_VER must match "a" value of BEAKER_PE_VER from a previous run. But this doesn't seem to say that.